CONTRIBUTING.md

```diff
@@ -206,24 +198,35 @@ The following checklist is used when reviewing pull requests
 ### General (all submissions)
 
 - Commit messages are useful and properly formatted
+- No sensitive personally identifiable information (PII) is included
 - Unit tests validate the code submission
 - One commit, or at most only a handful.  More than five commits should
   be heavily questioned
 
+#### About sensitive PII
+- To find out more about sensitive PII, refer to
+  [Sensitive PII versus non-sensitive PII](https://www.ibm.com/topics/pii#Sensitive+PII+versus+non-sensitive+PII)
```

Comment:
Remove this line as the link is from IBM and it is just IBM's opinion.

Reply:
Replaced link with https://www.investopedia.com/terms/p/personally-identifiable-information-pii.asp#toc-sensitive-vs-nonsensitive-pii